Android/app/src/main/java/io/github/project_travel_mate/utilities/CurrencyConverterAdapter.java

```diff
@@ -77,7 +77,7 @@ public void onBindViewHolder(@NonNull ViewHolder holder, int position) {
                 CurrencyConverterGlobal.global_country_name = mListCurrencyNamesFiltered.get(position).shortName;
                 CurrencyConverterGlobal.country_id = mListCurrencyNamesFiltered.get(position).abbreviation;
 
-            } else { // Is This is for world clock
+            } else { // Is This is for world Clock
```

Comment:
revert this

Reply:
I have changed it, you can see in commits.